README.md

```diff
@@ -2,5 +2,140 @@
 
 Monorepo containing CodersCamp internal application and its website.
 
+## Project requirements
```

Comment:
🤔 You should probably merge the main branch, as this documentation shouldn't be here anymore

Reply:
Done